### PR TITLE
Add fastify example for file uploads

### PR DESCRIPTION
**Pull request by zanechua**
Adding documentation for Fastify as requested in https://github.com/apollographql/apollo-server/issues/4975

### CODE REVIEW

File: docs/source/data/file-uploads.md [hunk 1]
```diff
@@ -86,3 +86,102 @@ async function startServer() {
 
 startServer();
 ```
+
+The following is an example for Fastify with `apollo-server-fastify`
+
+
+```js
+const { ApolloServer, gql } = require('apollo-server');
```
Comment (glasser) [inline docs/source/data/file-uploads.md:94]: Have you tested this code? I'd expect this line to say `apollo-server-fastify`.

**Comment by zanechua** [comment]
Tested and updated the example.

**Comment by glasser** [comment]
Thanks!

**Comment by zanechua** [comment]
@glasser 

Should this commit be in main instead?

i realised that this commit is missing from the main branch and the docs still have not been updated.

**Comment by glasser** [comment]
Oops, good catch! This was an old PR from when release-3.0 was the active dev branch. I'll cherry-pick it over and roll back the release branch.

**Pull request merged**